src/src/watchesdlg.cpp

```diff
@@ -758,16 +765,19 @@ void WatchesDlg::OnPropertyRightClick(wxPropertyGridEvent &event)
     if (prop && prop->GetLabel()!=wxEmptyString)
     {
         wxMenu m;
-        m.Append(idMenuRename, _("Rename"), _("Rename the watch"));
-        m.Append(idMenuAddDataBreak, _("Add Data breakpoint"), _("Add Data breakpoing"));
+        m.Append(idMenuRename,              "Rename",                "Rename the watch");
```

Comment:
I don't why you did this, but all the changes below are wrong. You've broken the translation of the menu items.
Also aligning this code would be reverted, by me. I don't align code.
This code is made to fit in 100 characters, which you've also reverted.

Reply:
Also I won't spend any time reviewing anything else before the PR31 (Ticket 1117 - Auto detect compiler startup) are approved.
One more thing: Don't remove wxEmptyString at will everywhere you see it. Keep your changes minimal, please. This helps me find the actual changes and review them.